mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -117,16 +117,33 @@ view: +subscriptions {
     sql: 1;;
   }
 
-  dimension: months_retained {
+  dimension: months_active {
     type: number
+    # month is timezone sensitive, so use localized datetime to calculate months
     sql: mozfun.norm.subscription_months_renewed(
-      -- month is timezone sensitive, so use localized datetime to calculate monthly retention
       DATETIME(${subscription_start_raw}, ${plan_interval_timezone}),
       DATETIME(${end_raw}, ${plan_interval_timezone}),
       ${billing_grace_period}
     );;
   }
 
+  dimension: current_timestamp_limit {
+    hidden: yes
+    description: "Maximum possible timestamp for which data is available. Assumes ETL is complete up to CURRENT_DATE."
+    sql: DATETIME_SUB(TIMESTAMP(CURRENT_DATE), INTERVAL 1 MICROSECOND);;
+  }
+
+  dimension: current_age_in_months {
+    description: "Number of months since subscription start date for which data is available. Assumes ETL is complete up to CURRENT_DATE."
```

Comment:
This dimension is valid for inactive accounts too, right?

Reply:
```suggestion
    description: "Number of months after subscription start date, regardless of whether the subscription was active or not."
```